core/vote/vote_pool.go

```diff
@@ -28,11 +26,6 @@ const (
 )
 
 var (
-	localCurVotesCounter    = metrics.NewRegisteredCounter("curVotes/local", nil)
```

Comment:
why these metrics are removed?

Reply:
We have gauge metrics for the same type so I remove the counter metrics